src/traces/heatmap/hover.js

```diff
@@ -26,6 +27,10 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode, contour)
         y = cd0.y,
         z = cd0.z,
         zmask = cd0.zmask,
+        zmin = trace.zmin,
```

Comment:
Don't scope variables if you're only going to use them once. The block below could be rewritten as:

```js
   var dummyAx = {
        type: 'linear',
        range: [trace.zmin, trace.zmax],
        hoverformat: trace.zhoverformat,
        _separators: trace._separators
   };
   var zLabel =  Axes.tickText(dummyAx, zVal, 'hover').text;
```